src/Sylius/Behat/Context/Transform/TaxonContext.php

```diff
@@ -41,6 +41,7 @@ public function __construct(TaxonRepositoryInterface $taxonRepository)
      * @Transform /^"([^"]+)" parent taxon$/
      * @Transform /^parent taxon to "([^"]+)"$/
      * @Transform /^taxon with "([^"]+)" name/
+     * @Transform /^main taxon (?:|should be )"([^"]+)"$/
```

Comment:
Do we need all these different patterns? Can't we use `:taxon` for generic steps as we do for instance in [ProductContext](https://github.com/michalmarcinkowski/Sylius/blob/654c168118797d7a42f405a29f322f3a71c366f0/src/Sylius/Behat/Context/Transform/ProductContext.php#L40-L40)?